src/plugins/intel_cpu/src/emitters/plugin/x64/jit_eltwise_emitters.cpp

```diff
@@ -1576,10 +1576,27 @@ void jit_power_static_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs,
             }
         }
     } else if (std::floor(power) == power && power != 0) {
-        int ipower = std::abs(static_cast<int>(power));
+        int64_t ipower = std::abs(static_cast<int64_t>(power));
```

Comment:
Shouldn't we update aarch64 emitter as well?

Reply:
Thanks Dmitry for the comment! I've checked that the aarch64 emitter is already correct. https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/src/emitters/plugin/aarch64/jit_eltwise_emitters.cpp#L1659-L1671
BTW, I've also aligned x64 algorithm with aarch64, which is more concise.